### PR TITLE
updated CLI docs for salt-ssh

### LINKED ISSUE #44313: salt-ssh: --user option missing from the cli documentation
### Description of Issue/Question
`--user` not on https://docs.saltstack.com/en/latest/ref/cli/salt-ssh.html

### Steps to Reproduce Issue
```bash
root@master:/vagrant# salt-ssh --help | grep user
    --user=SSH_USER     Set the default user to attempt to use when
```

### Versions Report
```yaml
Salt Version:
           Salt: 2017.7.1

Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.4.2
      docker-py: Not Installed
          gitdb: 0.6.4
      gitpython: 1.0.1
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.3
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.12 (default, Nov 19 2016, 06:48:10)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: 0.9.0
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: Ubuntu 16.04 xenial
         locale: UTF-8
        machine: x86_64
        release: 4.4.0-92-generic
         system: Linux
        version: Ubuntu 16.04 xenial
```

### PR DESCRIPTION
**Pull request by rossengeorgiev**
### What does this PR do?

* Adds missing options from the `salt-ssh` cli docs
* Updates order and sections of exists options

### What issues does this PR fix or reference?
Resolves #44313

### CODE REVIEW

File: doc/ref/cli/salt-ssh.rst [hunk 1]
```diff
@@ -19,13 +19,17 @@ Salt SSH allows for salt routines to be executed using only SSH for transport
 Options
 =======
 
-.. option:: -r, --raw, --raw-shell
+.. program:: salt
```
Comment (rallytime) [inline doc/ref/cli/salt-ssh.rst]: `salt` --> `salt-ssh`

File: doc/ref/cli/salt-ssh.rst [hunk 2]
```diff
@@ -53,38 +57,117 @@ Options
...
+   One or comma-separated list of extra Python modulesto  be included
+   into Thin Salt.
+
+.. option:: -v, --verbose
+
+   Turn on command verbosity, display jid.
+
+.. option:: -s, --static
+
+   Return the data from minions as a group after they all
```
Comment (rallytime) [inline doc/ref/cli/salt-ssh.rst]: This description isn't quite complete. Maybe missing `return` at the end of the sentence?

**Review by rallytime — CHANGES_REQUESTED** [review]
Thank you for updating these docs @rossengeorgiev. I had two small comments.

File: doc/ref/cli/salt-ssh.rst (hunk 1 above)
Comment (rossengeorgiev) [reply doc/ref/cli/salt-ssh.rst]: Didn't even notice that, I only moved that line around.

**Review by rallytime — APPROVED** [review]

**Pull request merged**